rcl/include/rcl/node.h

```diff
@@ -562,7 +562,8 @@ rcl_get_disable_loaned_message(bool * disable_loaned_message);
  * must register rcl_node_type_description_service_handle_request or a custom callback
  * to handle incoming requests, via that client's executor/waitset capabilities.
  *
- * This will initialize the node's type cache, if it has not been initialized already.
```

Comment:
Note that this is not true.  The type cache is actually initialized when the node is initialized.

Reply:
i might be mistaken... I think this is correct in `rcl` aspect? client library `rclcpp` needs to call this function to initialize type description service? in client library `rclcpp`, this is not true.

Reply:
This call will initialize the service, but the cache is already created in `rcl_node_init` here: https://github.com/ros2/rcl/blob/45fc9520216013007c2d4f3bd69d778af9051aa3/rcl/src/rcl/node.c#L278-L283